### ISSUE TITLE
Indentation incorrect for multiline expression in an assignment after the declaration that contains a null coalesce operator

### ISSUE THREAD

**Issue by eygraber** [detekt/detekt#3863]
Using Detekt 1.17.1. Not sure if this is part of the ongoing indentation issues with KtLint.

```
fun test() {
  val a: String = ""

  val someTest: Int?

  someTest =
    a
      .toIntOrNull()
      ?: 1
}
```

The Indentation rule is triggered by the third to last and second to last lines, and auto correct changes it to:

```
fun test() {
  val a: String = ""

  val someTest: Int?

  someTest =
    a
    .toIntOrNull()
    ?: 1
}
```

It seems like it's caused by a combination of the `val` (or `var`) not being on the same line as the assignment together with the null  coalescing operator.

```
  Indentation:
    active: true
    autoCorrect: true
    indentSize: 2
    continuationIndentSize: 4
```

**Comment by schalkms** [comment]
This rule is part of ktlint. Detekt provides a wrapper for it.
Could you please try to reproduce this issue with _plain_ ktlint?

**Comment by eygraber** [comment]
This is happening in plain ktlint ([detekt-3863.zip](https://github.com/detekt/detekt/files/6652536/detekt-3863.zip)). Should I move this over to the ktlint issues?

**Comment by BraisGabin** [comment]
Yes, please.

**Comment by eygraber** [comment]
https://github.com/pinterest/ktlint/issues/1165
